docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md

```diff
@@ -0,0 +1,509 @@
+
+## Getting started with {% include product %} Desktop
+
+The **{% include product %} Desktop** app sits at the heart of our integrations. It is a software application that extends the {% include product %} experience onto your local machine, and makes it easy to access {% include product %} data, launch applications, and run pipeline integration tools directly from your machine.
+
+**Running Toolkit Classic or a Custom Setup?**  
+
+The {% include product %} Integrations install automatically. If you are an advanced client wanting to install the **Classic** Toolkit configuration or you have a **Custom** Toolkit setup that you want to install via the **Desktop Setup Wizard** , please head over to the [administrator's guide](https://developer.shotgridsoftware.com/8085533c/?title=ShotGrid+Integrations+Admin+Guide) to learn how to do this.
+
+### Installation of Desktop
+
+You can download {% include product %} Desktop under the Apps menu.
+
+![shotgun-desktop-app-01.png](./images/sa-integrations-user-guide-shotgun-desktop-app-01.png)
+
+Follow the prompts and choose whether to download Desktop for Mac, Windows, or Linux.
+
+![download-desktop-02.png](./images/sa-integrations-user-guide-download-desktop-02.png)
+
+### Logging in
+
+The first time you open Desktop, you'll be prompted by your browser about certificates. We do this in order to connect {% include product %} with the Desktop app. If you’re having trouble getting certificates set up, take a look at our troubleshooting docs for [Firefox](https://developer.shotgunsoftware.com/d4936105/) and [Chrome and other browsers](https://developer.shotgunsoftware.com/95518180/) .
+
+![guide-desktop-cert-window-03.png](./images/sa-integrations-user-guide-desktop-cert-window-03.png)
+
+You will only need to update your security certificate list once. Please note that only Mac and Windows show a prompt for certificates, not Linux.
+
+Once that's set up, you'll need to log in to your {% include product %} site.
+
+If you go through a proxy, you will have to do some additional configuration. See instructions on how to do that in our [Admin Guide](https://developer.shotgridsoftware.com/8085533c/?title=ShotGrid+Integrations+Admin+Guide#toolkit-configuration-file) .
+
+#### Authentication and browser integration
+
+Desktop provides services that allow the {% include product %} web application to interact with your computer. This enables features like [local file linking](topicid=SG_Administrator_ar_data_management_ar_linking_local_files_html) and software launching. For security reasons, you must be logged into Desktop as the same user on the same {% include product %} site as you are in the browser. If you are logged in as a different user or on a different site, Desktop will prompt you to switch accounts and if you do it will restart and prompt for the appropriate credentials.
```

Comment:
Update this link to: https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html

Reply:
Link updated